src/fn/fn-rating-indicator.scss

```diff
@@ -0,0 +1,129 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-rating-indicator;
+
+@mixin fn-rating-indicator-labels-after-checked() {
+  .#{$block}__input:checked ~ .#{$block}__input + .#{$block}__label {
+    @content;
+  }
+}
+
+.#{$block} {
+  @include fn-reset();
+
+  cursor: pointer;
+  height: 1.375rem;
+  width: fit-content;
+  padding: 0.125rem 0.25rem;
+  border-radius: $fn-border-radius-4;
+
+  @include fn-hover() {
+    box-shadow: $fn-shadow-grey-level-1;
+
+    @include fn-active() {
+      box-shadow: $fn-focus-outline-shadow-grey;
+    }
+  }
+
+  @include fn-active() {
+    box-shadow: $fn-focus-outline-shadow-grey;
+  }
+
+  @include fn-focus-within() {
+    outline: none;
+    box-shadow: $fn-focus-outline-shadow-grey;
+  }
+
+  @include fn-disabled() {
+    pointer-events: none;
+    opacity: $fn-disabled-opacity;
+  }
+
+  @include fn-readonly() {
+    @include fn-hover() {
+      box-shadow: $fn-shadow-grey-level-1;
+
+      @include fn-active() {
+        box-shadow: $fn-shadow-grey-level-0;
+      }
+    }
+
+    @include fn-active() {
+      box-shadow: $fn-shadow-grey-level-0;
+    }
+
+    @include fn-rating-indicator-labels-after-checked() {
+      &::before {
+        content: '\e065';
+        color: $fn-color-grey-400;
+      }
+    }
+  }
+
+  @include fn-display() {
+    pointer-events: none;
+
+    .#{$block}__label::before {
+      color: $fn-color-grey-400;
+    }
+  }
+
+  &__container {
+    @include fn-reset();
+  }
+
+  &__input {
+    @include fn-reset();
+    @include fn-set-square(1.125rem);
+
+    z-index: -1;
+    opacity: 0.0001;
+    position: absolute;
+  }
+
+  &__label {
+    @include fn-reset();
+    @include fn-flex-center();
+    @include fn-set-square(1.125rem);
+    @include fn-set-margin-right(0.125rem);
+
+    &::before {
+      content: '\e065';
```

Comment:
can we move this to a variable?

Reply:
All these hardcoded values will be replaced by svg-s when Deno's PR is merged.